src/common/options.cc

```diff
@@ -1636,6 +1636,18 @@ std::vector<Option> get_global_options() {
     .set_description("how long the monitors wait before considering fully-healthy PGs as evidence the stretch mode is repaired")
     .set_min(1.0),
 
+    Option("mon_stretch_pool_size", Option::TYPE_UINT, Option::LEVEL_DEV)
+    .set_default(4)
+    .add_service("mon")
+    .set_description("")
+    .set_min_max(3, 6),
+
+    Option("mon_stretch_pool_min_size", Option::TYPE_UINT, Option::LEVEL_DEV)
+    .set_default(2)
+    .add_service("mon")
+    .set_description("")
+    .set_min_max(2, 4),
```

Comment:
could we avoid hardcoding these limits? In some disaster recovery cases we may need to reduce the size/min_size of the pools temporarily, and it'd be nice not to require a new build to do so.

Reply:
These are just the defaults that get set when you go in to stretch mode; you can still set a specific pool however you want later when doing recovery.

That said the recovery out of stretch mode is going to be complicated by the crush bucket distribution requirements, which I imagine will be more of a barrier than the min size.